.gitignore

```diff
@@ -42,8 +41,6 @@ tests.x
 *.xml
 untracked.info
 com.msg
-Katy_test/*
-KatyTest/*
 *~
 /utilities/run_process.conf
```

Comment:
`run_process.conf` is now located in `process/io/`, can this be changed here please